PontoCom/View/MainView.swift

```diff
@@ -86,10 +87,10 @@ struct MainView: View {
             return
         }
         
-        guard isLocationAllowed(location) else {
-            message = "Você está fora da zona permitida."
-            return
-        }
+//        guard isLocationAllowed(location) else {
+//            message = "Você está fora da zona permitida."
+//            return
+//        }
```

Comment:
Pode descomentar essas linhas

```suggestion
        guard isLocationAllowed(location) else {
            message = "Você está fora da zona permitida."
            return
        }
```